### PR TITLE
Recipe creation page

### PR DESCRIPTION
**Pull request by bfrieda**
related to issue #43 

image

image

### CODE REVIEW

File: preppal-fe/src/pages/new-recipe.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
+
+import { Container, Form, Button, Stack } from 'react-bootstrap';
+import { MdCancel } from 'react-icons/md';
+
+const LoginPage = () => {
+
+    return (
+        <Container className="d-flex justify-content-center">
+            <div className="w-100" style={{ maxWidth: '400px' }}>
+                <h2 className="text-center mb-4">Create Recipe</h2>
```
Comment (lik5) [inline preppal-fe/src/pages/new-recipe.tsx:10]: need to create more space between the nav bar and the top of the form

File: preppal-fe/src/pages/new-recipe.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                        <Stack direction="horizontal">
+                            <Form.Control type="cookTime"/>
+                            <Form.Select style={{ width: '50%'}}>
+                                <option value={1}>mins</option>
+                                <option value={2}>hours</option>
+                            </Form.Select>
+                        </Stack>
+                    </Form.Group>
+
+                    <Form.Group controlId="recipeNutritionFacts" style={{ paddingBottom: '24px' }}>
```
Comment (lik5) [inline preppal-fe/src/pages/new-recipe.tsx:82]: Let's remove this

**Review by lik5 — COMMENTED** [review]
We also need to make every input field larger, it looks kind of small.

**Comment by bfrieda** [comment]
> We also need to make every input field larger, it looks kind of small.

The screenshot was zoomed out to include everything, I updated the comment with how it looks now and not zoomed out
But I can change it if you still think it's too small

File: preppal-fe/src/pages/new-recipe.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+                            Submit
+                        </Button>
+                    </div>
+                </Form>
+            </div>
+        </Container>
+    );
+};
+
+export default LoginPage;
```
Comment (hwbit) [inline preppal-fe/src/pages/new-recipe.tsx:96]: Change the export default to NewRecipe.
Also add new routing to app.ts
I think we just add a link for this page to the navbar and do cleanup later.

**Comment by lik5** [comment]
> > We also need to make every input field larger, it looks kind of small.
> 
> The screenshot was zoomed out to include everything, I updated the comment with how it looks now and not zoomed out But I can change it if you still think it's too small

Can you make it so that the form is about this size 
Screenshot 2024-02-12 at 1 34 54 PM

**Comment by bfrieda** [comment]
> > > We also need to make every input field larger, it looks kind of small.
> > 
> > 
> > The screenshot was zoomed out to include everything, I updated the comment with how it looks now and not zoomed out But I can change it if you still think it's too small
> 
> Can you make it so that the form is about this size. Approximately like 2/3 of the screen width Screenshot 2024-02-12 at 1 34 54 PM



> > > We also need to make every input field larger, it looks kind of small.
> > 
> > 
> > The screenshot was zoomed out to include everything, I updated the comment with how it looks now and not zoomed out But I can change it if you still think it's too small
> 
> Can you make it so that the form is about this size. Approximately like 2/3 of the screen width Screenshot 2024-02-12 at 1 34 54 PM

I created a new issue #65 for this, since it's not a priority at the moment

**Review by lik5 — APPROVED** [review]

**Pull request merged**